test/python/test_pack.py

```diff
@@ -69,6 +76,9 @@ def deb_archive(tmpdir, light_project, request):
     cmd = [os.path.join(basepath, "cartridge"), "pack", "deb", project.path]
 
     if request.param == 'docker':
+        if project.deprecated_flow_is_used:
```

Comment:
I assume that this changes don't connected with current MR - move it to separate merge request, please

Reply:
See #168